Automattic/woocommerce-payments
# Add "Capital loans" overview page

Fixes https://github.com/Automattic/woocommerce-payments/issues/3453

#### Changes proposed in this Pull Request

- Added a dedicated "Capital loans" page to the main plugin navigation. That page will be displayed in the second-to-last position (before "Settings").
- That page and navigation item will only be displayed if the user has a Capital loan or had it in the past.
- The page will include a "Test mode" notice (similar to the other pages that can show test data), plus the "Active loan summary" card, plus a list of loans, displayed in tabular form.
- The list of loans won't have sorting or pagination. I think it's not needed, since merchants will have a handful of loans at most.
- Each row will be clickable, and it will take you to the "Transactions" page, filtered by the Loan ID.

#### Testing instructions

- Run https://github.com/Automattic/woocommerce-payments-server/pull/1650 (`add/all-loans-endpoint`) on the server.
- On an account that has no tapital loans, note that you don't see any change in the plugin's navigation.
- Switch to an account that has several capital loans. The more, the better.
- Note that now you see a "Capital Loans" navigation item. Click it.
- You should see a page with the Active Loan Summary card and a list of loans. Check the appearance of the page with the Figma design file.

-------------------
